packages/@react-spectrum/tag/docs/TagGroup.mdx

```diff
@@ -0,0 +1,224 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/tag';
+import {HeaderInfo, PropTable, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/tag/package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {Item, TagGroup} from '@react-spectrum/tag';
+```
+
+---
+category: Navigation
+keywords: [TagGroup, navigation]
```

Comment:
I think perhaps Collections or Status might be a better category since they are intended to categorize content. Not fully sold on either of those though.

Reply:
I agree. I'm more inclined to go with 'Status'. I put navigation because I saw it under 'Navigation' on the patterns page https://spectrum.corp.adobe.com/page/tag/

Reply:
@reidbarber should i go ahead and change it to collection or status? do you have a preference?

Reply:
Let's go with Status. We can update it before full release if needed.